### PR TITLE
[merge-scrolling]: Aggregate Widget Only scrolling - isolated from c...

### PR DESCRIPTION
**Pull request by sjunaid**
...ontent list scrolling

### CODE REVIEW

**Comment by mugurrus** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: client/app/scripts/superdesk-desks/desks.js [hunk 1]
```diff
@@ -119,6 +127,163 @@
...
+                                if (!_.isEqual(scope.items, scope.cacheNextItems)) {
+                                    scope.items = scope.items.concat(scope.cacheNextItems);
+                                }
+                            }, 100);
+
+                            api('archive').query(criteria)
+                            .then(function(items) {
+                                scope.cacheNextItems = items._items;
+                                scope.fetching = false;
+                            }, function() {
```
Comment (ahilles107) [inline client/app/scripts/superdesk-desks/desks.js]: i think that those empty error handling functions should be removed before merge. There are few places like this in this file.

File: client/app/scripts/superdesk-desks/styles/desks.less [hunk 2]
```diff
@@ -198,6 +198,8 @@
 				}
 				.stage-content {
 					padding: 2px;
+					overflow: scroll;
+                    height: 330px;
```
Comment (ahilles107) [inline client/app/scripts/superdesk-desks/styles/desks.less]: wrong indentation

File: client/app/scripts/superdesk-desks/desks.js [hunk 3]
```diff
@@ -119,6 +127,163 @@
...
+                        if (code === 40) {
+                            e.preventDefault();
+                            scope.move(DOWN, e);
+                        }
+                    });
+                });
+
+                scope.move = function (diff, event) {
+                    if (scope.selected != null && (scope.selected.task.stage === scope.stage)) {
+
```
Comment (ahilles107) [inline client/app/scripts/superdesk-desks/desks.js]: not needed empty line

**Comment by amagdas** [comment]
:+1:

**Pull request merged**